src/bun.js/webcore/streams.zig

```diff
@@ -1021,6 +1023,11 @@ pub const StreamResult = union(Tag) {
                 const value = result.toJS(globalThis);
                 value.ensureStillAlive();
 
+                switch (result.*) {
+                    .temporary => |*temporary| temporary.deinitWithAllocator(bun.default_allocator),
```

Comment:
temporary should mean that the StreamResult don't own the data and should be cloned, the owner (how created the StreamResult) should free it.